guides/11_Priority_Based_Downlink_Policy.md

```diff
@@ -59,6 +61,18 @@ Be aware of the following when calling this function:
 **Receiving notifications that a remote video was paused due to bandwidth constraint**
 If a remote video source is paused due to insufficient bandwidth, then the application will be notified through the existing notification mechanism by setting the VideoTileState to pausedDueToBandwidth. Note that there is already a member named `paused` in VideoTileState and that is used to track if the application paused the remote video.
 
+**Configure network event delays**
+Remote video source will be paused or unpaused based on bandwidth fluctuation. To configure the delay to pause/unpause, pass in two float values between _0 to 1_ (representing network issue response factor and recovery factor respectively) in initialization of `VideoPriorityBasedPolicyConfig`, and then assign the config to `VideoPriorityBasedPolicy`. The closer to 0 the value is, the smaller the delay is, and vice versa. Use the following setter to change the config dynamically:
+
+```ts
+setVideoPriorityBasedPolicyConfigs(config: VideoPriorityBasedPolicyConfig): void
```

Comment:
We should specifically mention on when developers should set this. What events should they listen too?